pywbem/_logging.py

```diff
@@ -14,404 +14,341 @@
+  and WBEM server. This logs the http request data and response data including
+  http headers and XML payload.
+
+The above logger names output log records when they are configured and
+activated. All pywbem loggers  log at the DEBUG log level and
+logging is activated for one or more :class:`~pywbem.WBEMConnection` instance
+objects.
+
+There are three steps to configuration of the pywbem loggers:
+
+* **Configure Python logger names** - Sets the Python logger parameters
+  (logging handlers, message formats, etc.). Either Python logging
+  methods or methods defined in :ref:`Logging configuration methods` may be
+  used to configure the loggers. The `dest` parameter of the pywbem methods
+  configures the loggers for either stream or file loggers and the `name`
+  parameter defines which of the pywbem loggers is being configured. This
+  configure step does not activate any pywbem logger. If no loggers are
+  configured, pywbem defaults to a logger with the Python
+  logging NullHandler.
+
+* **Set the log record detail level for pywbem loggers(Optional)** - Because
+  pywbem logs can generate large quantities of information, they allow the user
+  to control the quantity of information in each log record.  This is a pywbem
+  feature so it can only be configured using the pywbem methods defined in
+  :ref:`Logging configuration methods` using the `detail_level` parameter. The
+  `detail_level` possible values are defined in
+  :data:`~pywbem_logging.LOG_DETAIL_LEVELS`. The default for `detail_level` is
+  `all` which logs all if the data in the requests and responses if the
+  `detail_level` parameter is not set.
+
+* **Activate the pywbem loggers** - A pywbem logger is activated when it is
+  connected to one or more WBEMConnections so that log records will be generated
+  when any of the WBEMConnection methods that communicates with a WBEM server
+  is executed for that WBEMConnection. Pywbem separates the configuration
+  of the loggers from activation to allow logger configuration by either pywbem
+  methods or standard Python logging methods. Activation is controlled only by
+  the `connection` parameter of  the methods defined in
+  :ref:`Logging configuration methods` which either activates logging for
+  all (`connect=True`) subsequently created :class:`~pywbem.WBEMConnection`
+  objects or for a particular connection (`connect=conn` where conn is a
+  WBEMConnection object).
+
+.. _`Logging configuration methods`:
+
+Logging configuration methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The pywbem loggers may be configured through the following pywbem methods and
+must be used to set the log detail_level and activate the loggers.
+
+  * :meth:`~pywbem.WBEMConnection.configure_logger` - Configures and
+    optionally activates one or more of the loggers that log user-issued calls
+    to :class:~pywbem.WBEMConnection methods that drive WBEM operations and
+    pywbem http requests and responses. Thus, for example, the following
```

Comment:
On the first sentence "Configures and ...": 
* I would not repeat what the loggers log; this can be confusing if readers think this specifies some subset of the pywbem loggers. The method can simply configure and activate any of the pywbem loggers.
* This particular method can configure and activate only one logger, not one or more.
Suggest to change the sentence to:
"Configures and optionally activates a pywbem logger."

Reply:
DONE for first comment.  Not true for second since the keywork 'all' actually does both of them